### PR TITLE
Fractional Cover yamls

### PR DESCRIPTION
**Pull request by eloise-b**
Adding the yamls for fractional cover for LandSat for alchemist and the ODC product.

### CODE REVIEW

File: products/fc_ls.odc_product.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+---
+name: fc_ls
+description: Landsat Fractional Cover percentile 25 metre
```
Comment (alexgleith) [inline products/fc_ls.odc_product.yaml:3]: This is for percentiles

Comment (eloise-b) [reply products/fc_ls.odc_product.yaml:3]: I think I have the write one now!

File: products/fc_ls.odc_product.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+license: CC-BY-4.0
+
+metadata:
+  product:
+    name: fc_ls
+
+measurements:
+  - name: BS
+    units: "percent"
+    dtype: int16
```
Comment (alexgleith) [inline products/fc_ls.odc_product.yaml:15]: Should be `uint8` for all three

File: products/fc_ls.odc_product.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    nodata: -1
```
Comment (alexgleith) [inline products/fc_ls.odc_product.yaml:16]: Should be `255` for all three.

File: products/fc_ls.odc_product.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+name: fc_ls
+description: Landsat Fractional Cover
```
Comment (alexgleith) [inline products/fc_ls.odc_product.yaml:3]: Is this consistent with the Landsat WOfS description?

**Review by alexgleith — CHANGES_REQUESTED** [review]

File: products/fc_ls.odc_product.yaml (hunk 4 above)
Comment (eloise-b) [reply products/fc_ls.odc_product.yaml:3]: WOfS is: "Historic Flood Mapping Water Observations from Space"

**Review by alexgleith — APPROVED** [review]

**Pull request merged**